browsers-internal/releases/node-22/node22.15.1-bullseye-chrome136-ff138-edge/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build this image with command
+#   docker build -t cypress/browsers-internal:node22.15.1-bullseye-chrome136-ff138-edge --platform linux/amd64 .
+#
+FROM cypress/browsers-internal:node22.15.1-bullseye-chrome136-ff138
```

Comment:
out of curiousity where are we still using these internal images that ship with the image?